rasa/cli/x.py

```diff
@@ -422,8 +422,18 @@ def _get_credentials_and_endpoints_paths(
 
 
 def run_locally(args: argparse.Namespace):
-    # noinspection PyUnresolvedReferences
-    from rasax.community import local  # pytype: disable=import-error
+    try:
+        # noinspection PyUnresolvedReferences
+        from rasax.community import local  # pytype: disable=import-error
+    except ModuleNotFoundError:
+        from rasa.nlu.components import MissingDependencyException
```

Comment:
can we move the exception definition to `rasa.exceptions`? feels weird to import from `rasa.nlu`. And I'd move that import to the top level at the same time.

Reply:
good point, done :+1: